woeip/apps/data_collection/models.py

```diff
@@ -0,0 +1,118 @@
+from django.contrib.gis.db.models import LineStringField, PointField
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Organization(models.Model):
+    """An organization, e.g., WOEIP, OpenOakland"""
+    name = models.CharField(max_length=256)
+    website = models.CharField(max_length=256)
+    email = models.CharField(max_length=256)
+    contact = models.ForeignKey('Participant',
+                                related_name='contact', related_query_name='contact',
+                                on_delete=models.SET_NULL, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.name} <{self.email}>"
+
+
+class RoleEnum(models.Model):
```

Comment:
My understanding is that we decided against defined roles. Database permissions would be handled through the ```Organization``` model